bin/test.sh

```diff
@@ -63,12 +63,15 @@ configure_cli() {
 install_spark() {
     notify_github pending "Installing Spark"
 
-    # with universe server running, there are no longer enough CPUs to
-    # launch spark jobs if we give the dispatcher an entire CPU
-    # TODO: remove this?
-    echo '{"service": {"cpus": 0.1}}' > /tmp/spark.json
+    if [ "$SECURITY" = "strict" ]; then
```

Comment:
@nickbp When do we want to test with strict mode? Is it okay that it's not run by default?